src/models/thelio-mira-b4-n3/repairs.md

```diff
@@ -0,0 +1,613 @@
+
+5. Unplug all of the modular cabling from the back of the PSU.
+
+![PSU cabling](./img/psu-cables.webp)
+
+6. Remove/replace the power supply.
+    - The replacement PSU should be installed with the fan facing the bottom of the case.
+    - Place the PSU on the rubber post and hold it against the screwholes in the back of the case while attaching it.
+7. Connect the power cables to the new power supply and GPU (if installed), then reinstall the PSU brace, side brace, and top case.
+
+## Replacing the Thelio Io board:
+
+Named after Jupiter's moon Io, the Thelio Io daughterboard handles the front power button and fan control. If the Thelio Io board becomes defective, it can be replaced using the instructions below.
+
+**Part numbers:**
+- Thelio Mira B4-N3 uses [Thelio Io version 2](https://github.com/system76/thelio-io/tree/92c2a60b6d6acc0f9d51d819cba07ad28e2b03f3) ([PCB revision thelio_io_2.3](https://github.com/system76/thelio-io-hardware/tree/91814370002f762ac01cd1dd99dcbb43336352ab/pcb-thelio-io).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 30 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the Thelio Io board:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+    - The [CPU duct](#removing-the-cpu-duct) and [side brace](#removing-the-side-brace) can optionally be removed to provide easier access to the cabling.
+2. Use the cutout on the front of the chassis to unplug all cabling from the Thelio Io board.
+    - The topmost connector requires pulling the white tab while unplugging.
+
+![Thelio Io cabling cutout](./img/thelio-io-cabling-cutout.webp)
+
+3. Unscrew the two screws holding the daughterboard on from the right side of the chassis.
+
+![Thelio Io drive cage screws](./img/thelio-io-screws.webp)
+
+4. Screw in the new Thelio Io board and reconnect the wiring.
+
+### Thelio Io wiring guide:
+
+![Thelio Io wiring guide](./img/thelio-io-board.webp)
+
+1. When wiring the Thelio Io board, refer to the port labels and the following guide.
+    - `FANOUT1`/`CPU FAN` - to the top fan splitter (connecting to the [CPU fans](#replacing-the-cpu-fans)).
+    - `FANOUT4`/`AUX FAN` - to the bottom fan splitter (below the [CPU fan splitter](#replacing-the-cpu-fans)).
+    - `FANOUT2`/`INTAKE FAN` - to the [bottom case fan](#replacing-the-bottom-case-fan).
+    - `FANOUT3`/`GPU FAN` - to the [side brace](#removing-the-side-brace) fan splitter.
+        - May be disconnected by default if no side fan is ordered as factory configuration.
+    - `FANIN1`/`MOTHERBOARD FAN INPUT` - to the `CPU_FAN` header at the top right of the motherboard, highlighted cyan below.
+    
+    ![Motherboard CPU output header](./img/motherboard-fan-header.webp)
+    - `POWER1`/`POWER INPUT` - to the [power supply](#replacing-the-power-supply).
+        - The white plastic backing needs to be held away from the connector to unplug this cable from the Thelio Io board.
```

Comment:
"white plastic backing" seems like an awkward descriptor. Would "white plastic locking tab" or "white plastic latch" work better?

Reply:
I think "white plastic clip" is fine, and probably more natural than "backing."